src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/AbstractTask.groovy

```diff
@@ -70,4 +101,201 @@ abstract class AbstractTask extends DefaultTask {
         return (project.liberty.userDir == null) ? new File(installDir, 'usr') : new File(project.liberty.userDir)
     }
 
+    //Sets directories for the various configuration files
+    protected void setDirs(){
+        if(project.liberty.configDirectory != null && project.liberty.configDirectory.length() != 0){
+            configDirectory = new File(project.liberty.configDirectory)
+        }
+
+        if(project.liberty.configFile != null && project.liberty.configFile.length() != 0){
+            configFile = new File(project.liberty.configFile)
+        }
+        else{
+            configFile = new File("/src/main/liberty/config/server.xml")
+        }
+
+        if(project.liberty.bootstrapPropertiesFile != null && project.liberty.bootstrapPropertiesFile.length() != 0){
+            bootstrapPropertiesFile = new File(project.liberty.bootstrapPropertiesFile)
+
+        }
+        else{
+            bootstrapPropertiesFile = new File("/src/main/liberty/config/bootstrap.properties")
+        }
+
+        if(project.liberty.jvmOptionsFile != null && project.liberty.jvmOptionsFile.length() != 0){
+            jvmOptionsFile = new File(project.liberty.jvmOptionsFile)
+        }
+        else{
+            jvmOptionsFile = new File("/src/main/liberty/config/jvm.options")
+        }
+
+        if(project.liberty.serverEnv != null && project.liberty.serverEnv.length() != 0){
+            serverEnv = new File(project.liberty.serverEnv)
+        }
+        else{
+            serverEnv = new File("/src/main/liberty/config/server.env")
+        }
+    }
+
+    //Gets the loose bootstrap and jvm properties from the build file
+    protected void setProperties(){
+        if(project.liberty.bootstrapProperties != null && !project.liberty.bootstrapProperties.isEmpty()){
+            bootstrapProperties = project.liberty.bootstrapProperties
+        }
+        if(project.liberty.jvmOptions != null && !project.liberty.jvmOptions.isEmpty()){
+            jvmOptions = project.liberty.jvmOptions
+        }
+    }
+
+    private void copyFiles(File srcFile, File destFile){
+        def copyAnt = new AntBuilder()
+        copyAnt.copy(file:srcFile.getCanonicalPath(), tofile:destFile.getCanonicalPath(), overwrite:true)
+    }
+
+    /**
+     * @throws IOException
+     * @throws FileNotFoundException
+     */
+    protected void copyConfigFiles() throws IOException {
+
+        setDirs()
+        setProperties()
+
+        String serverDirectory = project.buildDir.toString() + "/wlp/usr/servers/" + project.liberty.serverName
```

Comment:
There should be a variable for the runtime location because if they are using `installDir`, the server might not be in the `buildDir` and might not start with `wlp`.  I didn't look but hopefully `installDir` is initialized properly when we do a repo install so you can use that (or some other variable) for all situations:  wlp on file system, wlp from repo, and wlp from maven coordinates (which Joseph is working on)